### PR TITLE
blueprints initial setup and Tangle storage blueprint

### PR DESCRIPTION
**Pull request by jakubcech**
Initial set up of the Blueprints folder, navigation, landing page section (with a whole 1 item at this point :)). 

- The blueprint document was broken down into different pages.
- Had to reuse some content so that user has context when necessary (PoC + Deployment instructions)
- Images should have a margin, they'll bleed right onto the content page border like this. 
- The flow may need some work. 
- Didn't touch the language. 
- Wrapped the unwrapped links.
- Will add other blueprints in the upcoming days. We need to first make sure this set-up works. 
- Only linking to deployment instructions, as they should live in the PoC repo anyway imo. 
- The overview to the whole Blueprints section needs way more love.

### CODE REVIEW

File: blueprints/0.1/tangle-data-storage/architecture.md [hunk 1]
```diff
@@ -0,0 +1,182 @@
+**This page describes an example architecture that can be used for solving the Tangle data storage use case.**
```
Comment (JakeSCahill) [inline blueprints/0.1/tangle-data-storage/architecture.md]: Needs a level 1 heading

File: blueprints/0.1/tangle-data-storage/architecture.md [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+
+**Disclaimer:** Running an open source project, like any human endeavor, involves uncertainty and trade-offs. We hope the architecture described below helps you to deploy similar systems, but it may include mistakes, and can’t address every situation. If you have any questions about your project, we encourage you to do your own research, seek out experts, and discuss with IOTA community.
+
+## Prerequesites for creating the project
```
Comment (JakeSCahill) [inline blueprints/0.1/tangle-data-storage/architecture.md]: Just 'prerequisites' is enough

File: blueprints/0.1/tangle-data-storage/architecture.md [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+The presented infrastructure makes use of the IOTA Tangle and an `InterPlanetary File System` (IPFS) node that you run yourself. The following image shows the main architecture components.
+
+![Data Storage PoC - IOTA/IPFS - Architecture](../data-storage-ipfs.png)
+
+The PoC implements two simple APIs:
+
+- storeFile
+- retrieveFile
+  
+## Store File
```
Comment (JakeSCahill) [inline blueprints/0.1/tangle-data-storage/architecture.md]: I think this page should only include conceptual information about the architecture we use in the blueprint and possible changes you may want to make in your own implementations.

The implementation (store files etc..) should go into a 'technical implementation' (something like that) article. Here, we can explain what the code does to help users to navigate the code on GitHub and learn where to make changes.

File: blueprints/0.1/tangle-data-storage/deployment-and-testing.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
+# Deployment and testing
```
Comment (JakeSCahill) [inline blueprints/0.1/tangle-data-storage/deployment-and-testing.md]: 'Deploy and test Tangle data storage'

This is a how-to guide, so the title should use infinitive verbs.

File: blueprints/0.1/tangle-data-storage/overview.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+- JS - Javascript
+- IRI - IOTA Reference Implementation, the SW written in JAVA that allows users to become part of the [IOTA](root://iri/0.1/introduction/overview.md) network as both a transaction relay and network information provider through the easy-to-use [API](root://iri/0.1/references/api-reference.md).
+- SHA256 - [Secure Hash Algorithm - 2](https://en.wikipedia.org/wiki/SHA-2)
+
+## Additional Resources
+
+- PoC Repository - https://github.com/iotaledger/poc-ipfs
+- iota.js Repository - https://github.com/iotaledger/iota.js 
+- NodeJS - https://nodejs.org/
+- IPFS - https://ipfs.io/
```
Comment (JakeSCahill) [inline blueprints/0.1/tangle-data-storage/overview.md]: Awesome 👍

File: blueprints/0.1/tangle-data-storage/architecture.md (hunk 3 above)
Comment (jakubcech) [reply blueprints/0.1/tangle-data-storage/architecture.md]: I agree that makes more sense. I think we can just keep it separate, so as not to get into too big a rewrite when migrating the blueprints here. I'll create an issue to address this in all the blueprints once they're all merged.

**Comment by jakubcech** [comment]
Thanks, addressed the comments. Made the section into "App Blueprints" instead of just "Blueprints" per Holger's input. It is a little more specific and the amazing benefit is that it fits into the menu width. 😬

**Comment by jakubcech** [comment]
This can be merged if there aren't any big obvious blockers that I missed. We can start with 2 blueprints and keep on adding. The only thing to consider is whether to have a blueprint section when there are only 2 items.

**Comment by obany** [comment]
A couple of items before the data storage blueprint is published:

- The github links point to poc-ipfs which is still a private repo.
- Any links to dag.sh need updating to iota.org as the demo page will be at https://ipfs.iota.org when published, the pipeline is already built just needs a final push for deployment

**Comment by jakubcech** [comment]
Do we plan on making the `poc-ipfs repo` public anytime soon?

**Comment by obany** [comment]
Is just needs reviewing by someone other than me :)

**Pull request merged**